configserver/src/main/java/com/yahoo/vespa/config/server/application/ZKTenantApplications.java

```diff
@@ -53,6 +60,8 @@ private ZKTenantApplications(Curator curator, Path applicationsPath, ReloadHandl
         this.directoryCache = curator.createDirectoryCache(applicationsPath.getAbsolute(), false, false, pathChildrenExecutor);
         this.directoryCache.start();
         this.directoryCache.addListener(this);
+        checkForRemovedApplicationsService.scheduleAtFixedRate(this::removeApplications, checkForRemovedApplicationsInterval.getSeconds(),
```

Comment:
FixedDelay to avoid potential queueing of tasks?

Reply:
Yeah, good point, will do